json-examples/tre-judgment-available.json

```diff
@@ -8,10 +8,8 @@
     "parentExecutionId" : null
   },
   "parameters" : {
-    "s3FolderUrl" : "https://aws/folder",
-    "judgmentDocx" : "out/docx",
-    "judgmentXml" : "out/xml",
-    "parserLog" : "out/log",
-    "reference" : "consignment-ref"
+    "reference": "(TDR-2023-AAA|FCL-???????) # opaque string: use originator to know where it came from, should be an acceptable filename",
+    "originator": "(FCL|TDR)",
+    "bundleFileURI": "https://s3.amazonaws.com/tre-output/TDR-2023-AAA.tar.gz?X-Amz-Security-Token=24db05… # download tar.gz",
+    "metadataFilePath": "metadata.json # location in bundle tar.gz # Metadata validation for metadata",
```

Comment:
just so I understand this comment, this filepath is used to validate the metadata file in the tarball is at the expected path?

Reply:
Ignore the comment about validation - it's just the location in the path to the metadata file in the tar
schema updated